jujugui/static/gui/src/app/components/deployment-flow/budget/test-budget.js

```diff
@@ -53,84 +60,90 @@ describe('DeploymentBudget', function() {
             'twelve-col': true
           }}
           clickable={false}
-          expanded={false}>
+          expanded={false}
+        >
           <div>
             <div className="four-col">
               <InsetSelect
                 disabled={false}
                 label="Budget"
                 onChange={wrapper.find('InsetSelect').prop('onChange')}
-                options={[{
-                  label: 'Big budget ($20)',
-                  value: 'Big budget'
-                }]} />
+                options={[
+                  {
+                    label: 'Big budget ($20)',
+                    value: 'Big budget'
+                  }
+                ]}
+              />
             </div>
             <div className="three-col">
               <span className="deployment-budget__increase-button">
                 <Button
-                  action={wrapper.find('Button').at(0).prop('action')}
+                  action={wrapper
+                    .find('Button')
+                    .at(0)
+                    .prop('action')}
                   disabled={false}
-                  type="base">
+                  type="base"
+                >
                   Increase budget
                 </Button>
               </span>
             </div>
-            <BudgetChart
-              budgets={budgets} />
+            <BudgetChart budgets={budgets} />
           </div>
           <div>
             <div className="deployment-budget__increase-form">
               <h4>Increase budget</h4>
-              <div className="two-col">
-                Credit limit: $100
-              </div>
-              <div className="ten-col last-col">
-                Available credit: $500
-              </div>
-              <div className="one-col">
-                Increase
-              </div>
+              <div className="two-col">Credit limit: $100</div>
```

Comment:
It isn't consistent here....sometimes it puts them in a line, sometimes it breaks them over multiple.... even if the line length isn't hit....maybe if the content IN the html is a statement `{}`